boards/pulga/include/board.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ * Copyright (C) 2022 Universidade de São Paulo
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dk
+ * @{
+ *
+ * @file
+ * @brief       Board specific configuration for the Pulga board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Sebastian Meiling <s@mlng.net>
+ * @author      Geovane Fedrecheski <geonnave@gmail.com>
```

Comment:
Just add yourself, no need to credit all the boilerplate.
This is more about who is responsible for the board.

Reply:
Ok.